### PR TITLE
Add HTTP Port 5778

### PR DESCRIPTION
**Pull request by Spellchaser**
https://www.jaegertracing.io/docs/deployment/#agent

```
$ grep -r 5775 *
all-in-one/jaeger-all-in-one-template.yml:                - containerPort: 5775
all-in-one/jaeger-all-in-one-template.yml:      port: 5775
all-in-one/jaeger-all-in-one-template.yml:      targetPort: 5775
jaeger-production-template.yml:          - containerPort: 5775
README.md:          - containerPort: 5775
README.md:The Jaeger Agent will then be available to your application at `localhost:5775`/`localhost:6831`/`localhost:6832`/`localhost:5778`.
$ grep -r 5778 *
all-in-one/jaeger-all-in-one-template.yml:                - containerPort: 5778
all-in-one/jaeger-all-in-one-template.yml:      port: 5778
all-in-one/jaeger-all-in-one-template.yml:      targetPort: 5778
jaeger-production-template.yml:          - containerPort: 5778
README.md:          - containerPort: 5778
README.md:The Jaeger Agent will then be available to your application at `localhost:5775`/`localhost:6831`/`localhost:6832`/`localhost:5778`.
```

Signed-off-by: Victor Lei <klei22@bloomberg.net>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,17 +113,18 @@ Assuming that your application is named `myapp` and the image is for it is `myna
...
           - containerPort: 6832
             protocol: UDP
+          - containerPort: 5778
+            protocol: TCP
           command:
           - "/go/bin/agent-linux"
           - "--collector.host-port=jaeger-collector.jaeger-infra.svc:14267"
 ```
 
-The Jaeger Agent will then be available to your application at `localhost:5775`/`localhost:6831`/`localhost:6832`.
```
Comment (Spellchaser) [inline README.md]: I'm not sure if this should be changed

**Comment by Spellchaser** [comment]
Addresses #91

**Comment by Spellchaser** [comment]
It's quite ridiculous to me that it's up to an external contributor to keep the license up to date... Not to mention I'm working behind proxies I don't control so mvnw doesn't work. 

Travis jobs have been broken since #89 at least

**Comment by jpkrohling** [comment]
I feel your pain!

> It's quite ridiculous to me that it's up to an external contributor to keep the license up to date

This is not always the case, but it's the first time this file was changed this year, so, the license checker requires this update.

> Travis jobs have been broken since #89 at least

Indeed, just wanted to avoid one more reason for it to be broken ;-)

@jkandasa, @kevinearls, do you know why the CI tests are broken?

**Comment by jpkrohling** [comment]
Looks like the tests are failing intermittently.

**Comment by jpkrohling** [comment]
Our integration tests are really far from being reliable...

**Comment by Spellchaser** [comment]
I'll push a no-op update to the other PR so it can rerun the CI tests

**Pull request merged**